e2e/cypress/plugins/client_request.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const axios = require('axios');
+
+module.exports = async ({data = {}, headers, method = 'get', url}) => {
+    let response;
+
+    try {
+        response = await axios({
+            data,
+            headers,
+            method,
+            url,
+        });
+    } catch (error) {
+        // If we have a response for the error, pull out the relevant parts
+        if (error.response) {
+            response = {
+                status: error.response.status,
+                statusText: error.response.statusText,
+                data: error.response.data,
+                itWasAnError: 'ffff',
```

Comment:
Whoops. That was left over from trying to debug. I basically spent far too long trying to figure out why
```js
try {
    doSomething();

    return true;
} catch (e) {}

return 'please return this';
```
kept returning true no matter what I did because I didn't notice the early return. I was trying everything to figure out what was going wrong.